Sources/MessagingPush/Extensions/NotificationCenterExtensions.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import UserNotifications
+
+extension UNNotificationResponse {
```

Comment:
Some comments in this file might be helpful

Reply:
Any comments in particular that you're wondering? 

I have some bias since I am familiar with the iOS UserNotifications framework so it's more difficult for me to guess what questions you may have.

Reply:
Not really anything particularly. But because I'm on the other side and less familiar iOS UserNotifications framework, its hard for me to pick how these checks actually work or what kind of extensions go to these files. The variable names are descriptive enough to guess, but if there is any additional info that you think might help the reader, that would be great.